### DISCUSSION TITLE
Trivy local image with podman

### DISCUSSION THREAD

**Discussion by Ladrillo84** [containers/podman#20600] [General]
Hello!

I want to transfer the trivy scanner to a local image with podman but I get an error.  The command I put is the following:

$ podman run -it --rm -v /run/podman/podman.sock:/run/podman/podman.sock <ID_IMAGE_TRIVY> image <ID_IMAGE_JENKINS> 

I hope you can advise or help me.

Thank you!

**Comment by rhatdan** [comment]
Most likely SELinux is blocking access to the socket.

`--security-opt label=disabled` will fix your problems.

**Reply by Ladrillo84** [reply]
Thank you for answer me. The same problem.....
